pyproject.toml

```diff
@@ -61,6 +61,8 @@ protobuf = "^4.21.6,<5.0.0" # chromadb currently fails on 5.0+
 opentelemetry-api = "1.25.0"
 opentelemetry-exporter-otlp-proto-grpc = "1.25.0"
 modal = "^0.64.145"
+pytest = "^8.3.3"
+pytest-asyncio = "^0.24.0"
```

Comment:
The pytest and pytest-asyncio dependencies are already declared in the group tool.poetry.group.test.dependencies.

We should not add them again.